crates/driver/src/util/bytes.rs

```diff
@@ -0,0 +1,30 @@
+/// A thin wrapper around a vector of bytes. Provides hex debug formatting.
+#[derive(Clone, Copy, PartialEq, Eq, Hash, Default)]
+pub struct Bytes<T>(pub T);
```

Comment:
It's common to not have bounds on the raw struct and only on the functions and traits that actually need the bound. Not sure about this case because it's purely a wrapper type but in general that's sensible.